src/markdown/lexicon/satellite-components/navigation/control-bar.md

```diff
@@ -14,8 +14,9 @@ Header is always fixed to the top of the window.
 The Header is used on almost every page in the system. The most common configuration is shown below:
 * Left area: Used for actions such as triggering the side menu and back navigation
 * Center area: Used for the page or application title and help
-* Right area: Used for actions. These are identified with an icon when possible and listed inside a kebab menu when an icon is not clear.
-    * These actions are application level actions
+* Right area: 
+    * Actions. These are identified with an icon when possible and listed inside a kebab menu when an icon is not clear.These actions are application level actions.
+    * User Sticker. It allows to display different options to which the user has access.
```

Comment:
User Sticker: It allows displaying different navigation options and actions to which the user has access.